mungegithub/mungers/blunderbuss.go

```diff
@@ -162,6 +162,19 @@ func selectMultipleOwners(potentialOwners weightMap, weightSum int64, count int)
 	return owners
 }
 
+func getPotentialOwnersHelper(author *string, feats *features.Features, files []*githubapi.CommitFile, issueNumber *int) (weightMap, int64) {
+	potentialOwners, weightSum := getPotentialOwners(*author, feats, files, true)
+	if len(potentialOwners) == 0 {
```

Comment:
@apelisse what do you mean here? return early if there are no approvers in the non-leaf call?  Then you'd have to make the non-leaf call first